### ISSUE TITLE
Outdated `2.2.0-alpha` spec in doc pages

### ISSUE THREAD

**Issue by michael-valdron** [devfile/api#1113]
## Which area this feature is related to?

/area documentation

## Which area is this documentation change is related to?

/area api

## Issue details

**What mistake did you find / what is missing in the documentation?**

The current doc pages includes the `2.2.0-alpha` spec which is no longer supported since the release of the `2.2.0` spec and has been superseded by `2.2.1-alpha`. The `2.2.0-alpha` spec should be removed from the doc pages.

**Target Date: 05-16-2023**

**Comment by michael-valdron** [comment]
Removal is completed, just waiting on additional reviews for a tied issue: https://github.com/devfile/api/issues/1057#issuecomment-1533732724

**Comment by michael-valdron** [comment]
https://github.com/devfile/api/issues/1057#issuecomment-1536824332

**Comment by michael-valdron** [comment]
Currently blocked by devfile/api#1057. Could cherry pick relevant changes out of https://github.com/devfile/devfile-web/pull/87 and into a new PR to unblock, otherwise will remain blocked until the completion of devfile/api#1057.

**Comment by michael-valdron** [comment]
No longer being blocked by #1057 and is now open for review in PR https://github.com/devfile/devfile-web/pull/89.
